packages/core/src/geometry/BufferSystem.ts

```diff
@@ -0,0 +1,149 @@
+import { System } from '../System';
+import { GLBuffer } from './GLBuffer';
+
+import type { Renderer } from '../Renderer';
+import type { IRenderingContext } from '../IRenderingContext';
+import type { Buffer } from './Buffer';
+
+/**
+ * System plugin to the renderer to manage buffers.
```

Comment:
I would describe this description as a _basic what_, which basically it the same as the name of the class.  I think it's always better to describe _how_ it works, _when_ to use it, and _why_ it exists.